### PR TITLE
News update - new article

### PR DESCRIPTION
**Pull request by SianHughes**

### CODE REVIEW

File: content/post/2018-03-14-opensourcing-in-enterprises-part1.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+The process of open sourcing can be very simple as well as difficult at times. It simply depends on the nature of your application. Some applications like 'website themes' usually does not require users to write their own config or link it to a database engine; hence they can be open sourced as is. But large enterprises does not deal with such applications. They often develop an application specific to their needs. So, they need to remove any organisation specific data from their code before open sourcing.
+
+We have recently helped one of our clients to open source an application which we also helped them to build. Here is a link to their public repository [https://github.com/iagcl/watchmen](https://github.com/iagcl/watchmen). We had seen many challenges during the process of publishing updates to the repo. Let's see what are those and how we have successfully solved them.
```
Comment (mngau) [inline content/post/2018-03-14-opensourcing-in-enterprises-part1.markdown]: "We recently supported one of our clients in open sourcing an application which we collaborated to build. Here is a link to their public repository [https://github.com/iagcl/watchmen](https://github.com/iagcl/watchmen). We saw many challenges during the process of publishing updates to the repo. Let's step through a few of them and how we solved them."

File: content/post/2018-03-23-automatically-compliant.markdown [hunk 2]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "Automatically Compliant"
-date: 2018-03-08T08:07:00+11:00
-draft: true
```
Comment (mngau) [inline content/post/2018-03-23-automatically-compliant.markdown]: Should the draft flag still be present?

Comment (SianHughes) [reply content/post/2018-03-23-automatically-compliant.markdown]: Has been removed

Comment (mngau) [reply content/post/2018-03-23-automatically-compliant.markdown]: Blog and News article will be published.

**Pull request merged**